### ISSUE TITLE
privileged/lidar-bucket/Shapefiles/goCD_Geodatabase.gdb

### ISSUE THREAD

**Issue by vermeulendivan** [dominodeorg/dominode#214]
Vector and raster data
Vector data has been converted to geoparquet
Raster data cannot be accessed because its a ESRI gdb, and ESRI has not released the encodings for this

The raster looks like its duplicates from other folders (waiting on feedback from client):

![image](https://github.com/dominodeorg/dominode/assets/79740955/a7d316a6-8933-4b94-b443-5c0fa1d5514c)

**Comment by EliVolsch** [comment]
Client confirmed the is no need to process this data, as the data is a duplicate of another folder group on the system.
